src/app/shared/models/waste-water.ts

```diff
@@ -41,6 +41,9 @@ export interface WasteWaterData {
 export interface ActivatedSludgeData {
     Temperature: number,
     So: number,
+    isUserDefinedSo: boolean,
```

Comment:
Test these fields with an old assessment, and update update-data.service if necessary. isUserDefinedSo may come up undefined/false and show invalid even if they had a value previously